test/standalone/Snow/parameterizations.jl

```diff
@@ -67,14 +72,13 @@ for FT in (Float32, Float64)
         )
 
         SWE = cat(FT.(rand(100) .+ 0.2), FT(0), dims = 1)
-        z = snow_depth.(SWE, ρ_snow, _ρ_l)
-        @test all(z .== SWE * _ρ_l ./ ρ_snow)
+        z = SWE * _ρ_l ./ ρ_snow
+        #@test all(z .== SWE * _ρ_l ./ ρ_snow)
```

Comment:
uncomment or remove?

Reply:
This will be dependent on our discussion about the above.